app/views/layouts/application.html.erb

```diff
@@ -1,16 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Not sure why this is being deleted, but doesn't sound like something you should do in this PR. If the PR title is "Config Active Storage" then we should only include the changes for that

Reply:
I keep this recommendation in mind for future pr